src/net/l4/tcp.c

```diff
@@ -100,55 +98,28 @@ static void tcp_get_now(struct timeval *out_now);
 
 /************************ Debug functions ******************************/
 #if !TCP_DEBUG
```

Comment:
Probably we can get rid of TCP_DEBUG macro since we can get the current log_level.